examples/todo-app/web/README.md

```diff
@@ -0,0 +1,27 @@
+# React + TypeScript + Vite
```

Comment:
Update the `README.md` with correct instruction on how to do the setup and installation including env file configuration